docs/Makefile

```
@@ -2,6 +2,8 @@ THEME := themes/gitea
 PUBLIC := public
 ARCHIVE := https://dl.gitea.io/theme/master.tar.gz
 
+HUGO_PACKAGE := github.com/gohugoio/hugo@v0.70.0
```

```suggestion
HUGO_PACKAGE := github.com/gohugoio/hugo@v0.81.0
```
This appears to be the first version with a clean (no replace directives) module.
https://github.com/gohugoio/hugo/blob/v0.80.0/go.mod
vs
https://github.com/gohugoio/hugo/blob/v0.81.0/go.mod

I haven't checked if this version will work, just noted that it's the minimum viable for `go run`